discord/abc.py

```diff
@@ -845,6 +845,11 @@ async def send(self, content=None, *, tts=False, embed=None, file=None,
 
             .. versionadded:: 1.4
 
+        message_reference: :class:`discord.MessageReference`
+            Message to which you are replying. If this is passed, then you should
```

Comment:
Looks like you are missing some words here

Reply:
Fixed, and made other aesthetic repairs where new features are documented.